brigadier/worker/email/tasks.py

```diff
@@ -33,16 +91,65 @@ def send_onboarding_mail(host, user_email):
 
     """
     send_mail(
-        'Welcome!',
-        f'Brigadier team welcomes you on board and we wish you a productive work.\n'
-        f'To help you achieve productivity take a tour:\n'
-        f'http://{host}'
-        f'{reverse("accounts:registration_activation_done")}',
+        _('Welcome!'),
+        _(
+            f'Brigadier team welcomes you on board and we wish you a productive work.\n'
+            f'To help you achieve productivity take a tour:\n'
+        ) + f'http://{host}'
+            f'{reverse("accounts:registration_activation_done")}',
+        None,
+        [user_email]
+    )
+
+
+@app.task(
+    max_retries=5,
+    default_retry_delay=60,
+    auto_retry_for=(ConnectionRefusedError,)
+)
+def create_overdue_notification_tasks():
+    """Task creates tasks for sending overdue notification emails.
+
+    """
+    overdue_emails = get_overdue_emails()
+
+    for overdue_data in overdue_emails:
+        send_overdue_notification_email.delay(
+            overdue_data.get('email'),
+            overdue_data.get('tasks_list_str')
+        )
+
+
+@app.task(
+    max_retries=5,
+    default_retry_delay=60,
+    auto_retry_for=(ConnectionRefusedError,)
+)
+def send_overdue_notification_email(user_email, overdue_tasks_list):
+    """Task sends overdue notification email.
+
+    """
+    send_mail(
+        _('Hello!'),
+        _(f'Seems that you have some overdue tasks here:\n')
+        + overdue_tasks_list,
         None,
         [user_email]
     )
 
 
+@app.on_after_finalize.connect
+def setup_periodic_tasks(sender, **kwargs):
+    """Periodic task for sending notifications about overdue tasks.
+
+    """
+    sender.add_periodic_task(
+        crontab(hour=12, minute=00, day_of_week='1-5'),
+        create_overdue_notification_tasks.s(),
+        name='Overdue notifications from Monday to Friday at 12:00.'
+    )  # pragma: no cover
+
```

Comment:
why don't you put all this periodic table initialization to in separate file? is there some issue ?
something like they introduced in [documentation](https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#crontab-schedules)

what if there is not only one periodic task but more that 20
their configuration will be scattered throughout the code
it'll be a nightmare to configure and control that schedule
so that's why I suggest to keep schedule table in separate config file,
especially when there is option for that